lib/graphql/relay/mutation.rb

```diff
@@ -199,6 +199,11 @@ def build_result(mutation_result, args, ctx)
           end
 
           if @wrap_result
+            unless mutation_result == nil || mutation_result.is_a?(Hash)
```

Comment:
It might just be me, but `unless` with `||` has always hurt my brain.

I think you can turn this line into:

```ruby
if mutation_result && !mutation_result.is_a?(Hash)
```